app/lib/sanitize_config.rb

```diff
@@ -50,13 +76,12 @@ module Config
         },
       },
 
-      protocols: {
-        'a' => { 'href' => HTTP_PROTOCOLS },
-      },
+      protocols: {},
```

Comment:
Won't it strip away whatever isn't supported by default?

Reply:
`spec/lib/sanitize_config_spec.rb:42` shows that it doesn't. `protocols` isn't by itself a whitelist, merely a definition of filters to be used. Nothing defined, no filters used.

Reply:
ok, this indeed works, I thought the config inherited from Sanitize's default one, but it's not the case